src/SDKs/SecurityCenter/Management.SecurityCenter/Generated/RegulatoryComplianceAssessmentsOperations.cs

```diff
@@ -122,7 +122,7 @@ public async Task<AzureOperationResponse<IPage<RegulatoryComplianceAssessment>>>
             }
             // Construct URL
             var _baseUrl = Client.BaseUri.AbsoluteUri;
-            var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/regulatoryComplianceStandards/{regulatoryComplianceStandardName}/regulatoryComplianceControls/{regulatoryComplianceControlName}/regulatoryComplianceAssessments").ToString();
```

Comment:
Were these files manually modified? It doesn't look like the changes were generated by Autorest